Ticketing/Ticketing.BAL/Contracts/IPaymentService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Ticketing.BAL.Model;
+
+namespace Ticketing.BAL.Contracts
+{
+    public interface IPaymentService
+    {
+        Task<PaymentStatusReturnModel> GetPaymentSatatusAsync(int paymentId);
+
+        Task CompletePaymentAsync(int paymentId);
+
+        Task FailedPaymentAsync(int paymentId);
```

Comment:
Shall be action, now statement. FailPaymentAsync or smth like that

Reply:
done